index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+
+    <header>
+      <h1 id="title"><title>Survey Form</title>Survey Form</h1>
+      <p id="description" > "What do you thing about our services"</p>
+    </header>
+    <main>
+      <form id="survey-form">
+l       <label id="name-label" for="name" >Please enter your name:</label>
+        <input id="name" type="text" required placeholder="Alejandro Maggioni">
+
+        <hr>
+        <label id="email-label" for="email" >Please enter your email:</label>
+        <input id="email" type="email" required placeholder="pizza@pizza.com">
+        <hr>
+
+        <label id="number-label" for="number" >Please enter your age</label>
+        <input id="number" type="number"  min="0"  max="120" placeholder="age" required=false>
+        <hr>
+
+        <label for="dropdown">Cake or Pizza or Sandwich</label>
+        <select id="dropdown">
+          <option>Cake</option>
+          <option>Pizza</option>
+          <option>Sandwich</option>
+        </select>
+        <hr>
+        <textarea name="name" rows="8" cols="80">Please write your reviews</textarea>
+
+        <input type="submit" id="submit">
+
+      </form>
+    </main>
+
+  </body>
+
+
+
+  </body>
```

Comment:
It looks like the body tag is duplicated. Please fix that. As additional comment, try not to let blank lines if do not be necessary.

Reply:
- [ ] Can you please specify why the change is required? Also, I felt the comment made sounded more like an order instead of suggestions/feedback. 😄 
For example:
This is an order: "Please fix that." or "Try not to leave blank lines"
This is a polite suggestion: "Kindly fix the issue as it may lead to unknown errors in the future." or "Kindly avoid leaving blank lines to make code clutter-free"  👍🏽 

 - [**OPTIONAL**] How about making the comments more human-friendly? I think it would be nice if you could add some emojis or gifs whenever possible in your comments to make it human-friendly 😃